releasenotes/notes/lo-freqs-circuit-c2ec161873eb4a68.yaml

```diff
@@ -0,0 +1,52 @@
+---
+features:
+  - |
+    The ability to pass qubit and measurement LO frequencies has now been added to circuit jobs at
+    both the job and experiment (individual circuit) level.
+
+    At the job level, the fields ``default_qubit_los`` and ``default_meas_los`` may be used as
+    arguents to ``execute`` for both circuit and pulse jobs. The fields are lists of floats
+    representing qubit drive and measure lo frequencies (in Hz) and are indexed by qubit number
+    (entry 0 is qubit 0, entry 1 is qqubit 1, etc...). The length of the list must be equal to the
```

Comment:
```suggestion
    (entry 0 is qubit 0, entry 1 is qubit 1, etc...). The length of the list must be equal to the
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/705ca3c97ae84c6f950f430f1eea576d4eb93e1c.